doc/source/Tutorials/GettingStarted/index.md

```diff
@@ -1,407 +1,499 @@
+
+With the textures assigned, the surface of all of Gaffy's geometry looks the same, because the material shader is applying to everything. To fix this, you should create an additional metallic shader and apply it selectively to different parts of the geometry.
+
+Begin by creating another shader:
+
+1. Create another Disney Material node.
+
+2. In the _Node Editor_, give the shader some metallic properties:
+	1. Set Metallic to `0.8`
+	2. Set Roughness to `0.4`
+
+3. With the new Disney Material node still selected, create another ShaderAssignment node
+
+4. Click and drag the new ShaderAssignment node onto the connector under the first ShaderAssignment node. The new ShaderAssignment node will come after the fist shader in the graph flow.
+
+    ![A second shader in the Graph Editor](images/graphEditorSecondShaderNodes.png "A second shader in the Graph Editor")
+
+The _Viewer_ will update to show the new shader.
+
+![The second shader, rendered](images/mainRenderTwoShaders.png "The second shader, rendered")
+
+You will immediately notice something is wrong: _all_ the geometry is metallic. The new shader has overridden the previous one. This is because:
+
+> Important :
+> The last ShaderAssignment node applied to geometry in a scene takes precedence over all others.
+
+### Filtering a Shader ###
+
+In order to selectively apply a shader to only certain parts of the scene's geometry, you will need to **filter** the shader assignment, using a filter node that selects parts of a scene by name:
+
+1. Create a PathFilter node (_Scene_ > _Filters_ > _PathFilter_).
+
+2. In the _Node Editor_, click ![the plus button](images/plus.png "Plus") next to _Paths_. This will add a new text field.
+
+3. Double-click the text field and type `/group/GAFFERBOT/C_torso_GRP/C_head_GRP/C_head_CPT/L_ear001_REN`. This is the full path to Gaffy's left ear.
+
+4. Connect the PathFilter node's _out_ plug to the filter input on the right hand side of the ShaderAssignment1 node's filter plug (yellow; on the right).
+
+    ![The connected PathFilter node](images/graphEditorPathFilterNode.png "The connected PathFilter node")
+
+Now when you check the render, you will see that the chrome shader is only applied to Gaffy's left ear. There are many other parts of Gaffy that could use the chrome treatment, but it would be tedious for you to manually enter multiple locations. There are two ways you can more easily add geometry to the filter: using text wildcards, and interacting directly with the geometry.
+
+
+#### Filtering Using Wildcards ####
+
+A wildcard tells text interpreters to take a value of any length where there is an asterisk `*`. They are useful if you know part of a path, but don't want to have to look up or type it in its entirety. Earlier, you used `/group/GAFFERBOT/C_torso_GRP/C_head_GRP/C_head_CPT/L_ear001_REN`, which only pointed to the left ear. You could apply the filter to _both_ ears by adding wildcards to the `/L_ear001_REN` location in the path.
```

Comment:
Do we need this text interpreter bit? I have trouble imaging someone who understands what a text interpreter is but needs an explanation about wildcards...

Reply:
Also, wildcards shouldn't really be encouraged on the grounds of not wanting to type the explicit path. They have a performance impact and should only be used when needed (eg to match multiple locations intentionally).

Reply:
I should probably drop this, yeah.